src/runtime_src/core/edge/user/system_linux.cpp

```diff
@@ -165,7 +195,7 @@ get_mgmtpf_device(device::id_type id) const
 
 void
 system_linux::
-program_plp(std::shared_ptr<device> dev, std::vector<char> buffer) const 
+program_plp(std::shared_ptr<device> dev, std::vector<char> buffer) const
```

Comment:
Not part of this PR, but still has to be fixed. Do not pass by value, especially buffer looks very inefficient, but also dev should be pass by const reference.

Reply:
Done, see above.

Reply:
Sorry, that's on me. I will take care of this asap